decompiler/config/jak2/all-types.gc

```diff
@@ -13798,6 +13878,11 @@
 ;; collide-target-h               ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+;; (deftype control-info (collide-shape-moving)
+;;   ()
+;;   :flag-assert #x4400001910
+;;   )
```

Comment:
![](https://media.giphy.com/media/E2USislQIlsfm/giphy.gif)

Reply:
Yay!

Reply:
On a positive note though, it appears to be 12588 bytes shorter than Jak 1's.